pkg/clients/grpc/connect.go

```
@@ -63,14 +63,19 @@
+type BaseConfig struct {
+	URL     string        `env:"URL"             envDefault:"localhost:7001"`
+	Timeout time.Duration `env:"TIMEOUT"         envDefault:"60s"`
+}
+
+type ManagerConfig struct {
+	BaseConfig
+	ClientCert   string `env:"CLIENT_CERT"     envDefault:""`
+	ClientKey    string `env:"CLIENT_KEY"      envDefault:""`
+	ServerCAFile string `env:"SERVER_CA_CERTS" envDefault:""`
+	BackendInfo  string `env:"BACKEND_INFO"    envDefault:""`
+	ClientTLS  bool   `env:"CLIENT_TLS"    envDefault:"false"`
```

golangci-lint

File is not `gci`-ed with --skip-generated -s standard -s default

(gci)